docs/content/guide/component.ngdoc

```diff
@@ -15,7 +15,7 @@ Advantages of Components:
 - simpler configuration than plain directives
 - promote sane defaults and best practices
 - optimized for component-based architecture
-- writing component directives will make it easier to upgrade to Angular
+- writing component directives will make it easier to upgrade to Angular 2
```

Comment:
"Angular" is used deliberately in order to keep in-line with our [branding guidelines][1].

[1]: http://angularjs.blogspot.gr/2017/01/branding-guidelines-for-angular-and.html